### PR TITLE
Feature/advantages

### PR DESCRIPTION
**Pull request by Anastasia-Katerinich**
Done

### CODE REVIEW

**Comment by Anastasia-Katerinich** [comment]
done

File: src/partials/advantages.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<section class="advantages">
+  <div class="container">
+    <h2 class="advantages-title visually-hidden">Заголовок</h2>
```
Comment (Serhii-Chekalov) [inline src/partials/advantages.html]: class="advantages__title

File: src/partials/advantages.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <ul class="advantages-list">
+      <li class="advantages-list__item">
```
Comment (Serhii-Chekalov) [inline src/partials/advantages.html]: class="advantages__item

File: src/partials/advantages.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <svg class="advantages-list__icon" width="100" height="102">
+          <use href="./images/icons.svg#man-group1"></use>
+        </svg>
+        <p class="advantages-list__text">Адаптированная авторская программа ВНО</p>
+      </li>
+      <li class="advantages-list__item">
+        <svg class="advantages-list__icon" width="100" height="102">
```
Comment (Serhii-Chekalov) [inline src/partials/advantages.html]: class="advantages__icon

**Pull request closed without merging**